### PR TITLE
[TTAHUB-2247] RTTAPA update table

### PR DESCRIPTION
**Pull request by thewatermethod**
## Description of change

Add a table to the RTR > RTTAPA page displaying recent communication logs where purpose = "Rttapa updates"

## How to test

Verify empty state of table. Create some communication logs. Verify that they display in the table accurately.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2247

### Every PR

- [ ] Meets issue criteria
- [ ] JIRA ticket status updated
- [ ] Code is meaningfully tested
- [ ] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [ ] API Documentation updated
- [ ] Boundary diagram updated
- [ ] Logical Data Model updated
- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions
- [ ] UI review complete

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [ ] Update JIRA ticket status

### CODE REVIEW

**Review by AdamAdHocTeam — COMMENTED** [review]
I don't see the RTTAPA Initial updates showing in the list?

image

File: frontend/src/pages/RecipientRecord/pages/__tests__/GoalsObjectives.js [hunk 1]
```diff
@@ -151,6 +154,8 @@ describe('Goals and Objectives', () => {
     // No Filters.
     const noFilterUrl = '/api/recipient/401/region/1/goals?sortBy=goalStatus&sortDir=asc&offset=0&limit=10';
     fetchMock.get(noFilterUrl, { count: 2, goalRows: noFilterGoals, statuses: defaultStatuses });
+
+    fetchMock.get('/api/communication-logs/region/1/recipient/401?sortBy=communicationDate&direction=desc&offset=0&limit=5&format=json&purpose.in[]=RTTAPA%20updates', { rows: [], count: 0 });
```
Comment (AdamAdHocTeam) [inline frontend/src/pages/RecipientRecord/pages/__tests__/GoalsObjectives.js]: Does it make sense to assert any data in the new table here?

File: src/routes/communicationLog/handlers.test.js [hunk 2]
```diff
@@ -171,6 +171,29 @@ describe('communicationLog handlers', () => {
...
+          sortyBy: 'communicationDate',
+          direction: 'asc',
+          limit: '20',
+        },
+      };
+      setTrainingAndActivityReportReadRegions.mockImplementation(() => Promise.resolve({}));
+      userById.mockImplementation(() => Promise.resolve(authorizedToReadOnly));
+      logsByRecipientAndScopes.mockImplementation(() => Promise.resolve([{ id: 1 }]));
+      await communicationLogsByRecipientId(mockRequest, { ...mockResponse });
+      expect(statusJson).toHaveBeenCalledWith([{ id: 1 }]);
```
Comment (AdamAdHocTeam) [inline src/routes/communicationLog/handlers.test.js:194]: Are we checking the limit wasn't exceeded here?

**Review by AdamAdHocTeam — COMMENTED** [review]
After last commit I don't see any communication's now...

File: src/routes/communicationLog/handlers.test.js (hunk 2 above)
Comment (thewatermethod) [reply src/routes/communicationLog/handlers.test.js:194]: no point, we mock the service

File: frontend/src/pages/RecipientRecord/pages/__tests__/GoalsObjectives.js (hunk 1 above)
Comment (thewatermethod) [reply frontend/src/pages/RecipientRecord/pages/__tests__/GoalsObjectives.js]: sure

**Comment by thewatermethod** [comment]
@AdamAdHocTeam 

> After last commit I don't see any communication's now...

Unable to duplicate. Did some cleaning up anyways, see if that works.

**Review by AdamAdHocTeam — APPROVED** [review]
Yea works after latest pull.

image

**Pull request merged**